Backend/Underdogs-Struggling/src/main/java/com/TheTroisMousquetaires/Underdogs/Struggling/payload/security/JWTAuthResponse.java

```diff
@@ -0,0 +1,27 @@
+package com.TheTroisMousquetaires.Underdogs.Struggling.payload.security;
+
+public class JWTAuthResponse {
+
+    private String accessToken;
+    private String tokenType;
+
+    public JWTAuthResponse(String accessToken) {
+        this.accessToken = accessToken;
+    }
+
+    public String getAccessToken() {
+        return accessToken;
+    }
+
+    public void setAccessToken(String accessToken) {
+        this.accessToken = accessToken;
+    }
+
+    public String getTokenType() {
+        return tokenType;
+    }
+
+    public void setTokenType(String tokenType) {
+        this.tokenType = tokenType;
+    }
```

Comment:
Most guides seem to hard code the tokentype to "Bearer". Will we be using a different tokentype?

Reply:
I guess not. I set tokentype to "Bearer" and removed the setter function.